wpilibNewCommands/src/main/native/cpp/frc2/command/MecanumFollowerCommand.cpp

```diff
@@ -0,0 +1,190 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2019 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#include "frc2/command/MecanumFollowerCommand.h"
+
+using namespace frc2;
+using namespace units;
+
+template <typename T>
```

Comment:
You can use wpi::sgn() in wpi/MathExtras.h instead. Although, this doesn't even appear to be used so you can remove it.